website/screens/components/date-input/code/DateInputCodePage.tsx

```diff
@@ -187,21 +187,17 @@ const sections = [
         ),
       },
       {
-        title: "Optional",
+        title: "Error handling",
         content: (
           <>
             <DxcText as="p">
-              To indicate that the date is optional, you can use the <Code>optional</Code> prop.
+              Example of error handling when it does not match the format. If the typed date does
```

Comment:
The first line "Example of error handling when it does not match the format." is a bit redundant with the second one "If the typed date does not match the defined format, an error will be displayed". Also, this can be a bit misleading, since the example manages any kind of errors, not just the format ones (for example, required).

Reply:
done